docs/content/configuration/indexing-service.md

```diff
@@ -306,6 +306,43 @@ Middle managers pass their configurations down to their child peons. The middle
 |`druid.worker.version`|Version identifier for the middle manager.|0|
 |`druid.worker.capacity`|Maximum number of tasks the middle manager can accept.|Number of available processors - 1|
 
+#### Processing
+
+Processing properties set on the Middlemanager will be passed through to Peons.
+
+|Property|Description|Default|
+|--------|-----------|-------|
```

Comment:
@josephglanville thanks for contribution, although the alignment is off, can you please fix it? 
you can see how it looks here https://github.com/josephglanville/druid/blob/1d560557fbd6bc02a05693d7cbf068158edb9b8f/docs/content/configuration/indexing-service.md
also if you are using IntelliJ it has some MD plugins to render the Md documents

Reply:
Fixed it, using IntelliJ and the current in-built rendering isn't working as it should. Do you have a recommendation for plugin?

Reply:
am using the native one `Markdown support for IntelliJ products` to, version `181.4203.550` seems working in my IDEA.

Reply:
I installed the `gfm` plugin which is adds another preview pane that renders as it would in Github.